### PR TITLE
Create Blog “why-is-redfish®-different-from-other-rest-apis-part-2”

### PR DESCRIPTION
**Pull request by donzef**
Automatically generated by Netlify CMS

### CODE REVIEW

**Review by choukden — CHANGES_REQUESTED** [review]
Hi Francois,

a few comments for you to take into consideration,

As for the bullet item list, you may want to use the style below to format the bullet list items in "standards" text.

<style>
li {
   font-size: 27px;
   line-height: 33px;
   max-width: none;
}
</style>

Just put the style at the top of your blog post. The bullet list will then be rendered as the same font as the rest of the content. 
In the CMS editor, on the right, you will see the font larger than the rest of the content. This is normal. No problem with the font. the final rendering will be correct at the time of publishing of the post.

You may want to apply the same on Part 1 of the series.

Thanks,
Denis

**Review by drensing — COMMENTED** [review]
Hi, Francois. Grammatically, pretty good! I'm teaching you a new lesson with this one - the proper use of R-balls.

But I have to say that the introduction was confusing to me and the summary was lacking a little in why this is a benefit.

Remember that our readers don't always read Part 1s and Part 2s consecutively. They might just stumble up Part 2 as the first one they've seen. You'll want to make sure you hook them in with each blog post, providing them with enough context to want to read more.

Sorry it took me a little while to get to this. Love seeing your submissions!
Dale

File: content/blog/why-is-redfish®-different-from-other-rest-apis-part-2.md [hunk 1]
```diff
@@ -0,0 +1,169 @@
...
+
+## Actions
+
+Redfish® resources support the GET request to retrieve their current state. Modifications or deletions can be performed to certain resources using POST, PUT, PATCH and DELETE requests. Until then, nothing exceptional in the world of REST APIs, except perhaps, that it is possible to retrieve the exhaustive list of possible requests on a given resource, by consulting the `Allow` header of GET request responses. Refer to the <a href="https://developer.hpe.com/blog/why-is-redfish®-different-from-other-rest-apis-part-1" target="_blank">Allowed requests</a> paragraph in part 1.
```
[the remark this answers is not in the document]
Comment (donzef) [reply content/blog/why-is-redfish®-different-from-other-rest-apis-part-2.md]: Removed "Until then": this was a bad translation from French.

File: content/blog/why-is-redfish®-different-from-other-rest-apis-part-2.md [hunk 2]
```diff
@@ -0,0 +1,169 @@
...
+
+However, other types of operations are difficult to model with classic HTTP requests. For example, it is impossible to "read" the server's power button to know its status! Asking for the "return to factory settings" of a storage controller may require additional parameters like the preservation (or not) of existing logical volumes.
```
[the remark this answers is not in the document]
Comment (donzef) [reply content/blog/why-is-redfish®-different-from-other-rest-apis-part-2.md:21]: Hopefully addressed.

**Comment by donzef** [comment]
Thanks Dale for the R-ball rule. It is clear now!

May be a quick final review with focus on the intro and the summary, to make sure I did not introduce other typos. Thanks.

**Pull request merged**